### PR TITLE
Add flow setup

### PR DESCRIPTION
**Pull request by dreampulse**
# Definition of Done

- [ ] The code does at least what is defined in the ticket and does not affect any other functionality
- [ ] The PR has a meaningful title
- [ ] There is a link to the issue-id
- [ ] The PR has been reviewed
- [ ] There are no 'hacks' or shortcuts refactoring is prefered

## Tests
- [ ] Reasonably tested with high code coverage (unit/integration)

##	Testability
- [ ] All exported functions are testable
- [ ] There is no code execution in the global scope
- [ ] Default exports are avoided
- [ ] The code is concise, expressive and readable
- [ ] The code is functional and doesn’t use class syntax

## React components are
- [ ] Stateless and use pure rendering functions
- [ ] Are documented and tested in storybook
- [ ] Typed-checked
- [ ] The Sass is stickily using the BEM-Convention
- [ ] For each react component exists exactly one Sass-File
- [ ] The is no container styled
- [ ] There are no console errors or unwanted console logs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,12 +16,14 @@
...
+    "test:flow": "flow",
     "test:dev": "NODE_ENV=test mocha --opts test/mocha.opts ./test/unit/ ./test/integration",
     "test:dev:watch": "NODE_ENV=test mocha --opts test/mocha.opts ./test/unit/ ./test/integration/ --watch",
-    "posttest": "npm run test:lint",
+    "posttest": "npm run test:lint && npm run test:flow",
     "changes": "./bin/update-changelog.js ./ --dry-run",
     "prerelease": "npm run test && ./bin/update-changelog.js ./",
-    "release": "./bin/bump-release.js ./"
+    "release": "./bin/bump-release.js ./",
+    "postinstall": "flow-typed install"
```
Comment (jhnns) [inline package.json]: `postinstall` wird afaik nicht ausgeführt, wenn man lokal `npm install` ohne Argumente ausführt. Der `postinstall` Hook ist dafür gedacht, wenn das Paket von NPM installiert wird: https://docs.npmjs.com/misc/scripts

Ich glaube `prepare` ist das, was du suchst.

Comment (dreampulse) [reply package.json]: Joah.. ich hatte das in nem später pr gefixed... mom.

Comment (dreampulse) [reply package.json]: Ich habs nach `update-types` umbenannt, da man das ja eigentlich manuell ausführen muss

Comment (dreampulse) [reply package.json]: Gegen `postinstall` spricht auch, dass `flow-typed install` sau langsam ist

**Pull request merged**